### DISCUSSION TITLE
Room temperature sensor / hidden remote?

### DISCUSSION THREAD

**Discussion by mptr8956** [JanM321/esphome-lg-controller#45] [General]
Hi! I am using MT09R ceiling mounted ACs and want to use this project to completely hide the remote inside the AC (theres enough room for that in the ceiling mounted versions). 

Question: If I build & connect it: Is it still using the temperature sensor inside the AC or does it take the temparature from a sensor on the new circuit board and do I need a separate temperature sensor somewhere in the room then?

Thanks in advance!

**Comment by JanM321** [comment]
It can use either the temperature sensor inside the AC or an external temperature sensor from Home Assistant. This is determined by the "Internal Thermistor" option in HA (see screenshots in README).

**Comment by mptr8956** [comment]
thats perfect. thank you very much :)
